src/gol.js

```diff
@@ -1,34 +1,36 @@
-export default function next(input) {
+export const countLiveNeighbours = (input, i, j) => {
```

Comment:
I like the fact that you order the function in better way. So functions are defined before they are used.